### DISCUSSION TITLE
Show edit layout link from the entry edit page in CP

### DISCUSSION THREAD

**Discussion by erinathop** [ExpressionEngine/ExpressionEngine#2967] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When trying to edit a layout for a particular channel it can be a bit of a hassle to get to the edit layout screen. I think it would be a big quality of life improvement if we could access the edit layout screen from within an entry. Similarly to how it's been done in previous versions of EE. 

**Describe the solution you'd like**
A link somewhere on the edit entry page in the CP to quickly and easily navigate to the edit layout screen for the relevant channel

**Describe alternatives you've considered**
I'm not aware of any 3rd party alternatives, i've been getting to this page from the channels listing page in the CP

**Teachability, Documentation, Adoption, Migration Strategy**
Simply put, users will be able to use this by clicking the link to quickly get to the edit layout screen.

- [x] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
I think displaying the information that certain page is using custom publish layout (with optional link to edit) would be useful

@Yulyaswan I think we could display Layout icon with a tooltip like "this page is using <name> publish layout" either in breadcrumbs or next to "edit entry" title. Or maybe you'll get better ideas

**Comment by Yulyaswan** [comment]
Added a button with a link to edit the layout page, which opens the page in a new tab.
PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/4011
